omod/src/main/webapp/resources/jsonforms/HMIS-TB-003-TBClientCard-EnrollmentPage.json

```diff
@@ -0,0 +1,234 @@
+{
+  "name": "DS TB Enrollment Form",
+  "version": "4.0.0",
+  "description": "This is the enrollment form for a DS TB patient",
+  "encounterType": "334bf97e-28e2-4a27-8727-a5ce31c7cd66",
+  "encounter": "Ds TB Enrollment",
+  "uuid": "2e26cc12-df19-4588-afc0-38ea107769fe",
```

Comment:
Ensure that form uuid matches that of old form "b00452b6-bb7c-11ea-b3de-0242ac130004"